src/main/java/com/redhat/devtools/lsp4ij/server/definition/launching/UserDefinedEditorBehaviorFeature.java

```diff
@@ -69,4 +69,11 @@ public boolean isEnableSemanticTokensFileViewProvider(@NotNull PsiFile file) {
         // Note that this defaults to enabled for user-defined language server definitions
         return (editorSettings == null) || editorSettings.enableSemanticTokensFileViewProvider;
     }
+
+    @Override
+    public boolean isEnableDocumentSymbolsBreadcrumbsInfoProvider(@NotNull PsiFile file) {
```

Comment:
Feature flag accessor for user-defined language server definitions based on client config.